lib/core/configurator.c

```diff
@@ -251,6 +251,10 @@ static yoml_t *convert_path_config_node(h2o_configurator_command_t *cmd, yoml_t
         /* convert to mapping */
         yoml_t *map = h2o_mem_alloc(offsetof(yoml_t, data.mapping.elements));
         map->type = YOML_TYPE_MAPPING;
+        map->filename = node->filename != NULL ? strdup(node->filename) : NULL;
+        map->line = node->line;
+        map->column = node->column;
+        map->anchor = node->anchor != NULL ? strdup(node->anchor) : NULL;
```

Comment:
I'd suggest doing first initializing the all properties of `map` then set the necessary columns to avoid issues when a new property gets added and for the ease of reading; i.e.:

```c
*map = (yoml_t){YOML_TYPE_MAPPING};
if (node->filename != NULL)
    map->filename = h2o_strdup(node->filename).base;
...
```

Note also that you should use `h2o_strdup` in place of `strdup` for the same reasons we use `h2o_mem_alloc` in place of `malloc`.